pyproject.toml

```diff
@@ -38,7 +38,7 @@ requires-python = ">= 3.11, < 4"
 dependencies = [
   "frequenz-api-microgrid >= 0.17.1, < 0.18.0",
   "frequenz-channels >= 1.0.0-rc1, < 2.0.0",
-  "frequenz-client-base >= 0.8.0, < 0.10.0",
+  "frequenz-client-base >= 0.8.0, < 0.11.0",
```

Comment:
Maybe even version 0.12 :eyes:

Reply:
But I had to remove that commit because this update was already merged.